xbmc/settings/Settings.h

```diff
@@ -56,6 +56,7 @@ class CSettings : public CSettingsBase, public CSettingCreator, public CSettingC
   static constexpr auto SETTING_FILELISTS_SHOWEXTENSIONS = "filelists.showextensions";
   static constexpr auto SETTING_FILELISTS_IGNORETHEWHENSORTING = "filelists.ignorethewhensorting";
   static constexpr auto SETTING_FILELISTS_ALLOWFILEDELETION = "filelists.allowfiledeletion";
+  static constexpr auto SETTING_FILELISTS_CONFIRMFILEDELETE = "filelists.confirmfiledelete";
```

Comment:
To be consistant with naming, we should name this setting after what we have for allow file deletion!

```suggestion
  static constexpr auto SETTING_FILELISTS_CONFIRMFILEDELETION = "filelists.confirmfiledeletion";
 ```